demo/loadmore.html

```diff
@@ -0,0 +1,83 @@
+<html>
```

Comment:
I've always been meaning to fix the demo pages; if you'll indulge me in some suggestions:

- prelude should be `<!DOCTYPE html><html lang="en">`
- Should include `<meta charset="utf-8">`
- Title should be informative (e.g. `&lt;virtual-scroller> demo: load more` or similar)
- Personally I like omitting `<head>` and `<body>` tags (both open and close), as well as closing `</html>` tag
- Maybe externalize the CSS in its own file?

For an example of the the basic structure see e.g. https://glitch.com/edit/#!/boom-bath?path=tabindex.html:150:0

Reply:
I did all except the last 2